package.json

```diff
@@ -41,7 +41,7 @@
   ],
   "license": "MIT",
   "dependencies": {
-    "@adastradev/data-ingestion-sdk": "^1.0.1",
+    "@adastradev/data-ingestion-sdk": "^1.0.2-beta.1",
```

Comment:
Will need updated to the latest once the sdk is merged into master.

Reply:
done